plugins/inputs/all/all.go

```diff
@@ -159,6 +159,7 @@ import (
 	_ "github.com/influxdata/telegraf/plugins/inputs/rethinkdb"
 	_ "github.com/influxdata/telegraf/plugins/inputs/riak"
 	_ "github.com/influxdata/telegraf/plugins/inputs/riemann_listener"
+	_ "github.com/influxdata/telegraf/plugins/inputs/rocm_smi"
```

Comment:
I understand "rocm_smi" is the name of the program that gathers data from the GPU, but I don't think it's a name that people will recognize. Could we change the plugin name to include AMD GPU or Radeon instead?

Reply:
changed in `amd_rocm_smi`, can it work?

Reply:
In their docs, AMD seems use "AMD ROCm" and "ROCm" interchangeably, so "amd_rocm_smi" seems like a good name for the plugin.

Reply:
great, so I renamed.